docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -379,182 +1209,733 @@ If you run `kubectl get pods -n sample-domain1-ns --watch`, then you should see
+
+{{% notice note %}}
+Perform the steps in [Prerequisites for all domain types](#prerequisites-for-all-domain-types) before performing the steps in this use case.
+{{% /notice %}}
+
+{{% notice warning %}}
+The operator does not support all possible dynamic model updates. For model update limitations, consult [Runtime Updates]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}}) in the Model in Image user docs, and carefully test any model update, before attempting a dynamic update in production.
+{{% /notice %}}
+
+Let's go through the steps:
+
+1. _Ensure you have a running domain._
+
+    Make sure you have deployed the domain from the [Initial use case](#initial-use-case).
+
+1. _Obtain a data source model YAML._
+
+    Find or create a WDT model snippet for a data source, make sure that its target is set to `cluster-1`, and that its initial capacity is set to `0`.
+
+   The reason for the latter is to prevent the data source from causing a WebLogic Server startup failure if it can't find the database, which would be likely to happen because we haven't deployed one (unless you're using the `JRF` path through the sample).
+
+   Here's a handy data source model configuration that meets these criteria:
+
+   {{%expand "Click here to expand." %}}
 
    ```
-   Hello World, you have reached server managed-server1
+   # Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+   # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+   resources:
+     JDBCSystemResource:
+       mynewdatasource:
+         Target: 'cluster-1'
+         JdbcResource:
+           JDBCDataSourceParams:
+             JNDIName: [
+               jdbc/mydatasource1,
+               jdbc/mydatasource2
+             ]
+             GlobalTransactionsProtocol: TwoPhaseCommit
+           JDBCDriverParams:
+             DriverName: oracle.jdbc.xa.client.OracleXADataSource
+             URL: '@@SECRET:@@ENV:DOMAIN_UID@@-datasource-secret:url@@'
+             PasswordEncrypted: '@@SECRET:@@ENV:DOMAIN_UID@@-datasource-secret:password@@'
+             Properties:
+               user:
+                 Value: 'sys as sysdba'
+               oracle.net.CONNECT_TIMEOUT:
+                 Value: 5000
+               oracle.jdbc.ReadTimeout:
+                 Value: 30000
+           JDBCConnectionPoolParams:
+               InitialCapacity: 0
+               MaxCapacity: 1
+               TestTableName: SQL ISVALID
+               TestConnectionsOnReserve: true
+
    ```
+   {{% /expand%}}
+
+   You can cut and paste the above model snippet to a file and then use it for your configmap, or alternatively, you can use the same data source that's already there for you in `/tmp/mii-domain/model-configmaps/model.20.datasource.yaml`.
```

Comment:
i got confused here -- what am i meant to do with this? how to i create the CM?  i assume that you will tell me later...